DigitalLearningSolutions.Data.Tests/DataServices/CentresDataServiceTests.cs

```diff
@@ -119,13 +119,24 @@ public void GetCentreDetailsById_should_return_null_centre_logo_if_logo_is_empty
         }
 
         [Test]
-        public void GetCentreName_should_return_null_when_the_centre_does_not_exist()
+        public void GetCentreIPPrefix_should_return_empty_array_when_the_centre_does_not_exist()

```

Comment:
Was this a duplicate test or is github just being dumb?

Reply:
This was a duplicate test (albeit the name was different) - see this test above:
Get_centre_name_should_return_null_when_the_centre_does_not_exist()